javascript/ql/src/semmle/javascript/HtmlSanitizers.qll

```diff
@@ -0,0 +1,46 @@
+/**
+ * Provides classes for working with HTML sanitizers.
+ */
+import javascript
+
+/**
+ * A call that sanitizes HTML in a string, either by replacing
+ * meta characters with their HTML entities, or by removing
+ * certain HTML tags entirely.
+ */
+abstract class HtmlSanitizerCall extends DataFlow::CallNode {
+  /**
+   * Gets the data flow node referring to the input that gets sanitized.
+   */
+  abstract DataFlow::Node getInput();
+}
+
+private class DefaultHtmlSanitizerCall extends HtmlSanitizerCall {
```

Comment:
May be worth adding a comment (even though it's private), explaining that this models popular npm packages as well as home-made sanitizers.